splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -663,6 +667,13 @@ def generate(
     removed_list = _remove_listed_files(ignore_list)
     if removed_list:
         logger.info("Removed:\n{}".format("\n".join(removed_list)))
+
+    comparator = CodeGeneratorDiffChecker(
+        source, os.path.join(output_directory, ta_name)
+    )
+    comparator.find_common_files(auto_gen_ignore_list)
+    comparator.print_files(logger)
```

Comment:
okay, I just had this thought (I don't know if it makes sense) that maybe the display will be called inside the checking function? We would simplify build.py a bit